src/main/connection.cpp

```diff
@@ -218,6 +218,10 @@ shared_ptr<Relation> Connection::Values(const string &values, const vector<strin
 	return make_shared<ValueRelation>(context, values, column_names, alias);
 }
 
+shared_ptr<Relation> Connection::EmptyValues(vector<LogicalType> types, vector<string> names, const string &alias) {
+	return make_shared<ValueRelation>(context, types, names, alias);
```

Comment:
Is this helper method necessary? It seems to not be used outside of a C++ test.

Reply:
I'm removing this entire approach, now it's just:
```c++
unique_ptr<DuckDBPyRelation> DuckDBPyRelation::EmptyResult(const std::shared_ptr<ClientContext> &context,
                                                           const vector<LogicalType> &types, vector<string> names) {
	vector<Value> dummy_values;
	D_ASSERT(types.size() == names.size());
	dummy_values.reserve(types.size());
	D_ASSERT(!types.empty());
	for (auto &type : types) {
		dummy_values.emplace_back(type);
	}
	vector<vector<Value>> single_row(1, std::move(dummy_values));
	auto values_relation =
	    make_uniq<DuckDBPyRelation>(make_shared<ValueRelation>(context, single_row, std::move(names)));
	// Add a filter on an impossible condition
	return values_relation->Filter("true = false");
}
```